mods/cnc/maps/nod02a/map.yaml

```diff
@@ -0,0 +1,405 @@
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 4
+	Actor26: e1
+		Location: 58,38
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 1
+	Actor27: e1
+		Location: 56,38
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 3
+	Actor28: e1
+		Location: 56,38
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 2
+	Actor29: e1
+		Location: 58,38
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 2
+	Actor30: e1
+		Location: 58,38
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 3
+	Actor31: e1
+		Location: 47,53
+		Owner: GoodGuy
+		Health: 1
+		Facing: 0
+		SubCell: 4
+	Actor32: e1
+		Location: 49,54
+		Owner: GoodGuy
+		Health: 1
+		Facing: 224
+		SubCell: 1	
+	Actor35: e1
+		Location: 34,57
+		Owner: GoodGuy
+		Health: 1
+		Facing: 64
+		SubCell: 1
+	Actor36: e1
+		Location: 56,38
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 4
+	Actor37: e1
+		Location: 56,38
+		Owner: BadGuy
+		Health: 1
+		Facing: 0
+		SubCell: 1
+	waypoint27: waypoint
+		Location: 27,51
+		Owner: Neutral
+	waypoint26: waypoint
+		Location: 49,35
+		Owner: Neutral
+	waypoint10: waypoint
+		Location: 30,44
+		Owner: Neutral
+	waypoint9: waypoint
+		Location: 55,40
+		Owner: Neutral
+	waypoint8: waypoint
+		Location: 48,41
+		Owner: Neutral
+	waypoint7: waypoint
+		Location: 60,58
+		Owner: Neutral
+	waypoint6: waypoint
+		Location: 61,47
+		Owner: Neutral
+	waypoint5: waypoint
+		Location: 48,55
+		Owner: Neutral
+	waypoint4: waypoint
+		Location: 41,51
+		Owner: Neutral
+	waypoint3: waypoint
+		Location: 28,36
+		Owner: Neutral
+	waypoint2: waypoint
+		Location: 28,51
+		Owner: Neutral
+	waypoint1: waypoint
+		Location: 40,43
+		Owner: Neutral
+	waypoint0: waypoint
+		Location: 40,35
+		Owner: Neutral
+	Refinery: proc
+		Location: 30,38
+		Owner: GoodGuy
+		Health: 1
+		Facing: 0
+	Barracks: pyle
+		Location: 34,43
+		Owner: GoodGuy
+		Health: 1
+		Facing: 0
+	Powerplant: nuke
+		Location: 35,40
+		Owner: GoodGuy
+		Health: 1
+		Facing: 0
+	Yard: fact
+		Location: 33,37
+		Owner: GoodGuy
+		Health: 1
+		Facing: 0
+	Guard1: e1
+		Location: 42,39
+		Owner: GoodGuy
+		Health: 1
+		Facing: 64
+		SubCell: 0
+	Guard2: e1
+		Location: 38,38
+		Owner: GoodGuy
+		Health: 1
+		Facing: 64
+		SubCell: 2
+	Guard3: e1
+		Location: 35,49
+		Owner: GoodGuy
+		Health: 1
+		Facing: 96
+		SubCell: 0
+	Guard4: e1
+		Location: 34,48
+		Owner: GoodGuy
+		Health: 1
+		Facing: 96
+		SubCell: 3
+	Guard5: e1
+		Location: 39,40
+		Owner: GoodGuy
+		Health: 1
+		Facing: 64
+		SubCell: 1
+	Guard6: e1
+		Location: 40,56
+		Owner: GoodGuy
+		Health: 1
+		Facing: 96
+		SubCell: 4
+	Guard7: e1
+		Location: 38,54
+		Owner: GoodGuy
+		Health: 1
+		Facing: 32
+		SubCell: 3
+Smudges:
+
+Rules:
+	Player:
+		-ConquestVictoryConditions:
+		MissionObjectives:
+			EarlyGameOver: true
+	World:
+		-CrateSpawner:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		ObjectivesPanel:
+			PanelName: MISSION_OBJECTIVES
+		LuaScript:
+			Scripts: nod02a.lua
+	^Vehicle:
+		MustBeDestroyed:
```

Comment:
So think, you mean all traits that concern structures and units. So I removed them and it still works.